content/en/docs/collector/internal-telemetry.md

```diff
@@ -34,23 +36,26 @@ service:
       address: '0.0.0.0:8888'
 ```
 
-You can enhance the metrics telemetry level using the `level` field. The
-following is a list of all possible values and their explanations.
+You can adjust the verbosity of the Collector metrics output by setting the
+`level` field to one of the following values:
 
-- `none` indicates that no telemetry data should be collected.
-- `basic` is the recommended value and covers the basics of the service
-  telemetry.
-- `normal` adds other indicators on top of basic.
-- `detailed` adds dimensions and views to the previous levels.
+- `none`: no telemetry is collected.
+- `basic`: essential service telemetry.
+- `normal`: the default level, adds standard indicators on top of basic.
+- `detailed`: the most verbose level, includes dimensions and views.
 
-For example:
+Each verbosity level represents a threshold at which certain metrics are
+emitted. For the complete list of metrics, with a breakdown by level, see
+[Lists of internal metrics](#lists-of-internal-metrics).
+
+The default level for metrics output is `normal`. To use another level, set
+`service::telemetry::metrics::level`:
```

Comment:
Why not use YAML notation given that this is YAML?

```suggestion
`service.telemetry.metrics.level`:
```

Reply:
Sorry. The notation came from the developers, so I was trusting their judgment. I'll change all of the config paths to match this suggestion.